src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,328 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static int taskCount=0;
+
+    // method prints a horizontal line.
+    public static void displayLine(){
+        System.out.println("____________________________________________________________");
+    }
+
+    // method prints Duke greeting.
+    public static void greeting(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("You are entering the\n" + logo + "\nZone...\n");
+
+        displayLine();
+        System.out.println("Hey there! Duke here!");
+        System.out.println("How can I serve you today?");
+        displayLine();
+    }
+
+    // method prints Duke termination.
+    public static void goodbye(){
+        System.out.println("Goodbye. See you in the funny papers.");
+        displayLine();
+    }
+
+
+    // method prints taskList.
+    public static void printList(){
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++){
+            System.out.print(String.format("%d. ", i + 1));
+            System.out.println(taskList[i]);
+        }
+        displayLine();
+    }
+
+    public static boolean isDeadline(String userInput){
+        if(userInput.length()>=8){
+            // return true if first 8 letters of userInput spell "deadline", else false
+            return userInput.substring(0,8).equals("deadline");
+        }
+        return false;
+    }
+
+    public static boolean isTodo(String userInput){
+        if(userInput.length()>=4){
+            // return true if first 4 letters of userInput spell "todo", else false
+            return userInput.substring(0,4).equals("todo");
+        }
+        return false;
+    }
+
+    public static boolean isEvent(String userInput){
+        if(userInput.length()>=5){
+            // return true if first 5 letters of userInput spell "event", else false
+            return userInput.substring(0,5).equals("event");
+        }
+        return false;
+    }
+
+    public static String getTodoFromUserInput(String userInput){
+        return userInput.substring(5,userInput.length());
```

Comment:
Avoid complicated expressions. Consider breaking up the steps into separate methods and statements.